src/govuk/components/pagination/_index.scss

```diff
@@ -0,0 +1,170 @@
+@include govuk-exports("govuk/component/pagination") {
+  .govuk-pagination {
+    border-top: 1px solid $govuk-border-colour;
+    text-align: center;
+
+    @include govuk-media-query($from: tablet) {
+
+      // Hide whitespace between elements
+      font-size: 0;
+
+      // Trick to remove the need for floats
+      text-align: justify;
+
+      &:after {
+        content: '';
+        display: inline-block;
+        width: 100%;
+      }
```

Comment:
🤯 

It took me a while to work out how this trick works...  As I now understand it, this is so that the pagination list appears on the left and the results sentence on the right.

It works by using the "justify text" mode, and then setting the 2 elements as `inline-block`, so that the browser lays them out as if it was a line with 2 words on it. Only, the words in the last line of a justified paragraph aren't justified, so a third, invisible `inline-block` element is generated with a 100% wide width, forcing it to appear on its own line, making the 2 visible elements no longer be on the last line.

Does that sound about right? I think we’d need some comments to remind ourselves how this works...?

Incidentally, what’s the reason for not using floats (like the grid-columns)? Is it to enable the widths of the 2 elements to be flexible? If so, would another alternative be CSS table layout (`display: table-row` and `display: table-cell`)?

Another option might be to just stack them on the left, which might be better for screen magnifier users, and doesn’t look too bad to my eye...

<img width="626" alt="Screenshot 2021-11-18 at 23 11 31" src="https://user-images.githubusercontent.com/30665/142511456-d5ca1840-9dd0-4537-a93b-39ac463053a8.png">

Reply:
Good detective work! This part of the code was inherited from the MoJ Design System - I'd not stopped to figure out what it was doing. I'd have to get a second opinion about whether table layout could work too - I wonder if @adamsilver or @edwardhorsford have a view on the idea of a stacked layout?